src/components/scenes/ChangePinScene.tsx

```diff
@@ -90,24 +90,24 @@ const ChangePinSceneComponent = ({
             onChangePin={handleChangePin}
           />
         </EdgeAnim>
-        <SceneButtons
-          primary={{
-            label: mainButtonLabel,
-            onPress: handlePress,
-            disabled: !isValidPin
-          }}
-          animDistanceStart={1}
-        />
       </ScrollView>
+      <SceneButtons
+        absolute
```

Comment:
The PIN Scene is clearly able to show the Next button. No need to peg to the bottom. This scene also very static and never changes height. Pegging Next to the bottom makes the user have to tap Done, then tap Next

Reply:
An acceptable alternative is to remove the numpad Done button and automatically drop the keyboard when the user taps the 4th PIN.

Reply:
I _always_ prefer consistency. Will do the alternative.

Reply:
I see the auto-keyboard drop but not the remove of Done button.